brandfolder/brandfolder.py

```diff
@@ -37,7 +37,7 @@ def create_asset(self, attachments_data, section_key, **attributes):
         }
 
         res = self.client.post(f'/{self.resource_type}/{self.id}/assets', json=data)
-        return Asset(self.client, res['data'][0])
+        return Asset(self.client, data=res['data'][0])
```

Comment:
while it seems perfectly clear in this case, I would vote for naming all arguments for consistency

`return Asset(client=self.client, data=res['data'][0]`